gcloud/pubsub/client.py

```diff
@@ -50,6 +50,12 @@ def list_topics(self, page_size=None, page_token=None):
         See:
         https://cloud.google.com/pubsub/reference/rest/v1/projects.topics/list
 
+        Example:
+
+        .. literalinclude:: pubsub_snippets.py
+           :start-after: [START client_list_topics]
+           :end-before: [END client_list_topics]
```

Comment:
Embedding snippets in the method docstrings makes testing them infeasible:  moving them into the new, system-test-like file means that they can be actually exercised against the real API, which is the only way to ensure that the don't bitrot.

What evidence do we have that printing examples at the Python REPL prompt is important?  I don't expect that for any other Python library I use:  even the stdlib doesn't have that!  Instead, I expect to read the docs (online, normally).  Maybe we could look at producing [dash](https://kapeli.com/dash_ios)-style docsets, which can be consumed with [zeal](http://zealdocs.org/) (Linux), [velocity](http://velocity.silverlakesoftware.com/) (Windows), or [lovelydocs](http://lovelydocs.io/) (Android), to facilitate offline use of the docs.